release/mk-cfgman-docs.sh

```diff
@@ -6,14 +6,25 @@
 
 . ~/.server.config || exit $?
 
-# Known Bug:
-#  Does not remove references to directives dropped. This is
-#  an issue for branch 'master' directive erasure.
```

Comment:
Is not this bug still present? AFAICT, the DYN part of the loop iterates previously published `*.html` directive files and does not erase any stale ones.